locale/core.yml

```diff
@@ -164,6 +164,7 @@ core:
       view_last_seen_at_label: View user last seen time
       view_post_ips_label: View post IP addresses
       view_user_list_label: View user list
+      edit_user_label: Edit User
```

Comment:
The keys should be sorted alphabetically. Also, please don't use title case for the permission labels (i.e. change it to "Edit user").